### PR TITLE
Added Scheduler.py

### PR DESCRIPTION
**Pull request by matosBrent**
### Submission Checklist

- [x] Your pull request targets the `master` branch of the repository.
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).

#### Type of Change

- [ ] Bug fix
- [x] New implementation

### PR Description

Fixes #103

### CODE REVIEW

File: scripts/python/scheduler.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    #  extracting minutes out of the time
+            tempMinute = str(listTime[1])
+            minute = tempMinute[0:2]
+    #  extracting seconds out of the time
+            tempSecList = tempMinute.split(':')
+            tempSecString = str(tempSecList[1])
+            sec = tempSecString[0:2]
+            if( minute == '24' and sec == '01'):
+                if hour == '11':
+                    with open('C:\Users\harvey\Desktop\Prod.txt','r+') as f:
```
Comment (DipanshKhandelwal) [inline scripts/python/scheduler.py]: It has local links. Please update the code

File: scripts/python/scheduler.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                        f.write("Please send in the 11pm - 12am CST report")
+                    webbrowser.open_new_tab('C:\Users\harvey\Desktop\Prod.txt')
+                    time.sleep(1)
+                elif hour == '12':
+                    with open('C:\Users\harvey\Desktop\Prod.txt','r+') as f:
```
Comment (DipanshKhandelwal) [inline scripts/python/scheduler.py]: here

File: scripts/python/scheduler.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                        f.write("Please update the Client about the latest Web Server Report\n")
+                    webbrowser.open('C:\Users\harvey\Desktop\Prod.txt')
+                    time.sleep(1)
+                elif hour == '13':
+                    with open('C:\Users\harvey\Desktop\Prod.txt','r+') as f:
```
Comment (DipanshKhandelwal) [inline scripts/python/scheduler.py]: here

File: scripts/python/scheduler.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                        f.write("Please send in the 1am - 2am CST report and also update the TGT daily Stats report. The purge process starts at 1330hrs IST\n")
+                    webbrowser.open_new_tab('C:\Users\harvey\Desktop\Prod.txt')
```
Comment (DipanshKhandelwal) [inline scripts/python/scheduler.py]: and many places like this. Please update it.

**Review by DipanshKhandelwal — CHANGES_REQUESTED** [review]

**Comment by matosBrent** [comment]
Changes done @DipanshKhandelwal

**Review by DipanshKhandelwal — APPROVED** [review]
Thanks

**Pull request merged**